docs/content/commands/npm-explain.md

```diff
@@ -26,9 +26,8 @@ alias: why
 This command will print the chain of dependencies causing a given package
 to be installed in the current project.
 
-Positional arguments can be either folders within `node_modules`, or
-`name@version-range` specifiers, which will select the dependency
-relationships to explain.
+If one or more package specs are provided, then only packages matching
```

Comment:
This probably needs rewording still to retain the original concept of specifying folders inside node_modules.

Reply:
The examples show this behavior so I think this is ok as is.